code/modules/shuttle/on_move.dm

```diff
@@ -327,7 +327,7 @@ All ShuttleMove procs go here
 /obj/structure/ladder/afterShuttleMove(turf/oldT, list/movement_force, shuttle_dir, shuttle_preferred_direction, move_dir, rotation)
 	. = ..()
 	if (!(resistance_flags & INDESTRUCTIBLE))
-		LateInitialize()
+		LateInitialize(FALSE)
```

Comment:
Comment

Reply:
I assume you mean like `/* mapload = */`

Reply:
Yup